metricflow/plan_conversion/instance_converters.py

```diff
@@ -586,6 +593,47 @@ def transform(self, instance_set: InstanceSet) -> InstanceSet:  # noqa: D
         )
 
 
+class UpdateMeasureFillNullsWith(InstanceSetTransform[InstanceSet]):
```

Comment:
Not totally relevant to the PR, but this whole "instance converter" pattern is something I need to understand better... i.e., when is it appropriate to add a new instance converter vs. just putting that code in `dataflow_to_sql` or wherever it's relevant? Maybe a question for @tlento!

Reply:
yea abit hazy on that as well. I had it here just cause it felt cleaner. My take on this is that we would put it in the converter when we're mutating the instance spec set itself. But I guess there's also cases where we;re converting the spec set into select columns, etc...

Reply:
@courtneyholcomb The original context on that is - let's say we created a new type of instance and had to update the code base. How do we find all the cases that need to be updated? The thought was that centralizing that logic into these classes would make it easy to do so. We haven't had to add a new instance in a while though. However, `dataflow_to_sql` is already large so separating things into smaller functional chucks is preferred.